README.md

```diff
@@ -13,82 +14,80 @@ http://www.jinteki.net
 
 ## Card implementation status
 
-
 [Card rules implementation status](https://docs.google.com/spreadsheets/d/1ICv19cNjSaW9C-DoEEGH3iFt09PBTob4CAutGex0gnE/pubhtml)
 
 
-## Dependencies
+## Development
+### Quickstart
+
+Install [Leiningen](https://leiningen.org/) and [NodeJS](https://nodejs.org/en/download/package-manager/#macos).
+
+This project runs on Java 8. If you're on OSX or Linux, we recommend using
+[jenv](https://github.com/jenv/jenv/blob/master/README.md) to manage your java environment.
+
+You can check your setup by running
+
+    $ lein version # Your exact version numbers below may vary, but we expect Java 1.8.X
+    Leiningen 2.9.1 on Java 1.8.0_222 OpenJDK 64-Bit Server VM
+
+A quick way to get a MongoDB database set up is using [Docker](https://www.docker.com/):
+
+    $ docker run -d -p 27017:27017 --name netrunner -v netrunner-data:/data/db mongo
+
+Populate the database using:
 
-* [Leiningen](https://leiningen.org/) (version 2+)
-* [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)
+    $ lein fetch
 
+You can optionally pass `--no-card-images` if you don't want to download images from
+[NetrunnerDB](https://netrunnerdb.com/), as this takes a while. See `lein fetch help` for further options.
 
-## Installation
+To install frontend dependencies, run:
 
-Install frontend dependencies:
+    $ npm install -g bower
```

Comment:
What's your reasoning for changing the code formatting from using backticks to indentation?

Reply:
I think it makes the unrendered markdown a bit easier to read personally.